src/util/styles.ts

```diff
@@ -59,7 +59,7 @@ export function getSelectorSpecificity(selector: string) {
 		.replace(/[\*\s\+>~]/g, ' ')
 		.replace(/[#\.]/g, ' ');
 
-	[currentSelector, delta] = findSelectorMatch(currentSelector, elementRegex);
+	[currentSelector, delta] = findSelectorMatch(currentSelector, elementRegex); // lgtm [js/useless-assignment-to-local]
```

Comment:
why not just do `[_, delta] = findSelectorMatch(currentSelector, elementRegex);` instead of adding the comment to disable lgtm? would that fix it? @brettz9 @dangreen

Reply:
also if we have linting set up through our own repo, what else does lgtm provide beyond that? should we just get rid of it knowing that we have linting and code coverage built in already?

Reply:
As far as the specifics, yes, I could use that code instead. Since the function kept repeating the same pattern though, I wasn't sure if it might need to be added to again in the future. But sure, if you prefer that way, I can amend to use that instead.

As far as LGTM, good question... LGTM delves more deeply into the code structure in many cases than ESLint does. From my experience, it catches many things which ESLint doesn't and often things which are very important like security issues. However, ESLint is probably more comprehensive in what it catches (including styling consistency which LGTM only flags in certain cases), and ESLint also works locally, so I think they are complementary. LGTM doesn't look at coverage, so that's separate too.

Reply:
ok thanks for the detailed answer! we'll merge for now and can come back to this once we have eslint set up the way we want

Reply:
Ok, sure. Btw, if you were interested, you might want to use something like [`npm-check-updates`](https://github.com/tjunnone/npm-check-updates/) to bump the relevant deps/devDeps. I tried to do it but had trouble with the tests after installing with Yarn and building on my Mac (I had installed with npm and it worked ok).

Reply:
1) `yarn upgradeInteractive --latest`
2) We have dependabot